themes/codefor-theme/assets/scss/breakpoints/_base.scss

```diff
@@ -404,7 +416,7 @@ footer{
     }
   }
   details {
-    & > summary::-webkit-details-marker {
+    & > summary::marker {
```

Comment:
netify rendert da im chrome auf "Mitmachen" Pfeile in die liste,
laut webkit ist -webkit-details-marker debrecated und wird bei mir in allen Browsern richtig angezeigt (also ohne Pfeile)
/mitmachen/index.html